ci_scripts/ci_deploy.sh

```diff
@@ -28,8 +29,7 @@ if [[ ! -e ${BIN_DIR}/deploy_scripts_${DEPLOY_SCRIPT_VERSION} ]]; then
 fi
 
 # determine APP_PATH
-APP_VERSION=`cat version.properties | grep "version" | cut -d'=' -f2`
-APP_PATH="api/build/libs/$APP_NAME-$APP_VERSION.jar"
-export APP_PATH
+export APP_VERSION=`cat version.properties | grep "version" | cut -d'=' -f2`
+export APP_PATH="api/build/libs/$APP_NAME-$APP_VERSION.jar"
```

Comment:
These changes are to provide an APP_VERSION environment variable to the deploy script, which will then (in an upcoming PR) provide that environment variable to the application in cloud foundry (via `cf set-env`).